src/current/_includes/v25.2/leader-leases-node-heartbeat-use-cases.md

```diff
@@ -0,0 +1,7 @@
+For the purposes of [Raft replication]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft) and determining the [leaseholder]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-leaseholder) of a [range]({% link {{ page.version.version }}/architecture/overview.md %}#architecture-range), node health is no longer determined by heartbeating a single "liveness range"; instead it is determined using [Leader leases]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leader-leases).
+
+However, node heartbeats of a single range are still used to determine:
+
+- Whether a node is still a member of a cluster (this is used by [`cockroach node decommission`]({% link {{ page.version.version }}/cockroach-node.md %}#node-decommission)).
+- Whether a node is dead or not (in which case [its leases will be transferred away]({% link {{ page.version.version }}/architecture/replication-layer.md %}#how-leases-are-transferred-from-a-dead-node)).
+- To avoid placing replicas on dead, decommissioning or unhealthy nodes, and to make decisions about lease transfers.
```

Comment:
```suggestion
- Whether to avoid placing replicas on dead, decommissioning or unhealthy nodes, and to make decisions about lease transfers.
```

Reply:
went with "how to avoid", since there is no "whether" opposing case where we would want to put replicas on dead nodes etc